pyproject.toml

```diff
@@ -102,12 +102,11 @@ markers = [
 [tool.ruff]
 target-version = "py311"
 line-length = 120
-select = ["A", "ARG","ASYNC","BLE","C90", "E", "F", "I", "N", "UP032", "W"]
 exclude = ["notebooks/**", "**/*.ipynb"]
 
 [tool.ruff.lint]
-select = ["E", "F", "I"]
-ignore = []
+select = ["A", "ARG","ASYNC","BLE","C90", "E", "F", "I", "UP032", "W"]
+ignore = ["E741"]
```

Comment:
I find it useful to comment what the ignored rules are

Reply:
added explanation